libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Microsoft.AdaptiveDialog.schema

```diff
@@ -22,6 +22,14 @@
             "description": "Value that will be passed back to the parent dialog.",
             "default": "dialog.result"
         },
+        "dialogs": {
+            "type": "array",
+            "title": "Referenced dialogs.",
+            "items": {
+                        "type": "string"
+                     },
```

Comment:
Should have title and description.  Formatting is also off for this JSON.

Reply:
added title and description here.